### PR TITLE
framework/media: Alloc and init audio card info

### PR DESCRIPTION
**Pull request by junmin-kim**
Allocate and initialize audio_card_info at audio_manager_init.

### CODE REVIEW

File: framework/src/media/audio/audio_manager.c [hunk 1]
```diff
@@ -682,6 +681,33 @@ static audio_manager_result_t set_audio_volume(audio_io_direction_t direct, uint
...
+		return AUDIO_MANAGER_OPERATION_FAIL;
+	}
+
+	for (card_id = 0; card_id < CONFIG_AUDIO_MAX_INPUT_CARD_NUM; card_id++) {
+		pthread_mutex_init(&(g_audio_in_cards[card_id].card_mutex), NULL);
+	}
+
+	g_audio_out_cards = (audio_card_info_t *)calloc(CONFIG_AUDIO_MAX_OUTPUT_CARD_NUM, sizeof(audio_card_info_t));
+	if (g_audio_out_cards == NULL) {
+		return AUDIO_MANAGER_OPERATION_FAIL;
```
Comment (jeongchanKim) [inline framework/src/media/audio/audio_manager.c]: Is it okay not to free the g_audio_in_cards?

Comment (junmin-kim) [reply framework/src/media/audio/audio_manager.c]: I'll fix it

**Comment by seinfra** [comment]
Target : [79164d383937c50a9f309cbae9f5e3a08bdcc685] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [79164d383937c50a9f309cbae9f5e3a08bdcc685] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [2718633c9c2a868d09c48d06ded05841b60cdcc8] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [2718633c9c2a868d09c48d06ded05841b60cdcc8] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [91c33f9c867cb7460941508193a62db68ccc577c] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [91c33f9c867cb7460941508193a62db68ccc577c] - Code Rule Check (C++) OK.

File: framework/src/media/audio/audio_manager.c [hunk 2]
```diff
@@ -682,6 +681,35 @@ static audio_manager_result_t set_audio_volume(audio_io_direction_t direct, uint
 
 audio_manager_result_t audio_manager_init(void)
 {
+	static int am_initialized = 0;
+	int card_id;
+
+	if (am_initialized) {
+		return AUDIO_MANAGER_SUCCESS;
+	}
+
```
Comment (Taejun-Kwon) [inline framework/src/media/audio/audio_manager.c]: init -> find all cards -> malloc as same as founded card.

**Comment by seinfra** [comment]
Target : [460dd7c493f96387669fb63d716a9cd83e8fe311] -

**Comment by seinfra** [comment]
Target : [460dd7c493f96387669fb63d716a9cd83e8fe311] -

**Comment by seinfra** [comment]
Target : [35bbe4511c11d2c771b5048d4cc0a08286602fbb] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [35bbe4511c11d2c771b5048d4cc0a08286602fbb] - Code Rule Check (C++) OK.

**Pull request closed without merging**